test/e2e/templates_test.go

```diff
@@ -0,0 +1,138 @@
+package e2e
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	routeclient "github.com/openshift/client-go/route/clientset/versioned"
+
+	e2e "github.com/openshift/cluster-authentication-operator/test/library"
```

Comment:
sort imports?

Reply:
I think they are sorted, aren't they?
```
sys deps

outside deps

kube deps

openshift deps

local deps
```

Reply:
I was under the impression that imports must be sorted in the order

```
sys deps

github deps

kube deps
```

Didn't know that they could sorted this way too.